index.js

```diff
@@ -195,6 +202,12 @@ function isSourceElement(node) {
     case "TypeAliasDeclaration": // Typescript
     case "ExportAssignment": // Typescript
     case "ExportDeclaration": // Typescript
+    case "ObjectExpression": // JSON
```

Comment:
Good point, fixed in https://github.com/prettier/prettier/pull/2298/commits/29631046cddbfcd5cb38b94cc5b333be51fccb60

We can fix the other parsers separately.